app/forms/idv/inherited_proofing/base_form.rb

```diff
@@ -64,6 +64,11 @@ def user_pii
 
       attr_writer :payload_hash
 
+      # Return extra data used to instantiate the FormResponse
+      # returned upon #submit.
+      def extra
```

Comment:
Subclassing, mixins, oop in general will invariably invite _some_ measure of "indirection" unless you avoid these completely, which _could_ become a detriment.

I personally don't see this as a slippery slope. If someone needed to actually _use_ `extra`, they'd obviously have to look to the subclass, and it's plain as day what they'd need to do. These classes are _so_ small. 

I see this as a "pay me now, or pay me later" quandary. Whatever Form class we otherwise would come up with would certainly need to 1) be subclassed at some point, or 2)  duplicate the code because additional service providers json returned will certainly not conform to the json returned from other service providers. 

If it concerns you that much, I can remove it, and subclasses can simply override the `#submit` method.

Reply:
Addressed.